build/scripts/cache_images.sh

```diff
@@ -40,13 +40,13 @@ for image in "${images[@]}"; do
   image_dir="${RESOURCES_DIR%/}/${image_dir%/*}"
   mkdir -p "$image_dir"
 
-  cached_image="${image_dir%/}/${filename}"
+  cached_image="${image_dir%/}/${filename%\?*}"
```

Comment:
I guess `#` fragment also should be cared here. @amisevsk

Reply:
Thanks @monaka, I missed that. I've fixed it to strip both query and fragment components, and also to avoid an edge case where a url contains multiple `?` or `#` chars.

Reply:
I checked 2c34856 and LGTM.

Reply:
Thanks @monaka!